storage/src/vespa/storage/config/stor-communicationmanager.def

```diff
@@ -33,6 +33,8 @@ mbus.rpctargetcache.ttl double default = 600
 ## Any value below 1 will be 1.
 mbus.num_threads int default=4
 
+mbus.optimize_for enum {LATENCY, THROUGHPUT} default = LATENCY
```

Comment:
this feels a bit generic

If it only controls tcp_nodelay, we can just be more specific. If we add more code alternatives choosing between latency and throughput later we might not be able to test their effects separately, or they need more configuration than a single flag.

Reply:
I do not want to expose too much details. We should figure out a set that works for all.